src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Clearbooks\Labs\Toggle\Entity;
 
-interface ActivatableToggle
+interface ActivatableToggle extends Toggle
```

Comment:
yeah that was my vision for the toggle interfaces
lots of itty bitty interfaces that provide windows into a big object somewhere else